gap/semigroups/semifp.gi

```diff
@@ -541,3 +541,100 @@ IsCollsElms, [IsFpSemigroup, IsElementOfFpSemigroup],
 function(S, x)
   return SEMIGROUPS.ExtRepObjToWord(ExtRepOfObj(x));
 end);
+
+InstallMethod(IsomorphismFpSemigroup,
+"for an inverse partial perm semigroup",
+[IsPartialPermSemigroup and IsInverseActingSemigroupRep],
+function(M)
+  local add_to_odd_positions, S, SS, G, GG, s, g, F, rels, fam, alpha,
+  beta, lhs, rhs, map, H, o, comp, U, rhs_list, conj, MF, T, inv, rel, x, y, m,
+  i;
+
+  if not IsFactorisableInverseMonoid(M) then
+    TryNextMethod();
+  fi;
+
+  add_to_odd_positions := function(list, s)
+    local i;
+    for i in [1, 3 .. Length(list) - 1] do
+      list[i] := list[i] + s;
+    od;
+    return list;
+  end;
+
+  # FIXME: this shouldn't be necessary but for an error with
+  # MinimalFactorization
```

Comment:
Please file an issue for this error, or is this already resolved?

Reply:
I can't remember the issue now. It was something we noticed when writing code together. I am not sure if it was a bug or just that the format of the output didn't suit our purposes (e.g. due to how it dealt with inverses or the identity when factorizing).

My tests still pass when I remove the `rec(acting := false)`. MinimalFactorization uses `EN_SEMI_FACTORIZATION` so

Reply:
Aha, then this was fixed, it was Issue #424. Please remove the comment.

Reply:
Ah okay, great.

Reply:
Please also remove the unnecessary code.